tests/unit-core/test-api.c

```diff
@@ -709,12 +709,12 @@ main (void)
   if (jerry_feature_enabled (JERRY_FEATURE_PROXY))
   {
     jerry_value_t target = jerry_object ();
-    jerry_value_t handler = jerry_object ();
-    jerry_value_t proxy = jerry_proxy (target, handler);
+    jerry_value_t handler_ = jerry_object ();
+    jerry_value_t proxy = jerry_proxy (target, handler_);
```

Comment:
this isn't nice. what is the problem of cppcheck with handler?

Reply:
The problem was that is shadowed the `handler` function defined at line 64.
I agree its not a good name, it was just a placeholder.